### DISCUSSION TITLE
C2280 “Ort::Value::Value(const Ort::Value &)”

### DISCUSSION THREAD

**Discussion by yimingchao432** [microsoft/onnxruntime#18041] [Other Q&A]
std::vector<std::vector<float>> inputs;
std::vector<Ort::Value> ort_inputs;
//auto memory_info = Ort::MemoryInfo::CreateCpu(OrtArenaAllocator, OrtMemTypeDefault);
auto memory_info = Ort::MemoryInfo::CreateCpu(OrtDeviceAllocator, OrtMemTypeCPU);

for (size_t i = 0; i < num_input_nodes; i++) 
{
	Ort::Value input_tensor = Ort::Value::CreateTensor<float>(memory_info, inputs[i].data(),
		input_node_dims_sum[i], input_node_dims_vector[i].data(), input_node_dims_vector[i].size());
	//assert(input_tensor.IsTensor());
	ort_inputs.emplace_back(input_tensor);

}

**Comment by RyanUnderhill** [comment]
I'm not quite sure what your question is?

My guess is that you're trying to copy input_tensor and your last line should be: `ort_inputs.emplace_back(std::move(input_tensor))`
